test/cli_create_test.go

```diff
@@ -360,6 +361,71 @@ func (suite *PouchCreateSuite) TestCreateWithEnv(c *check.C) {
 	}
 }
 
+// TestCreateWithEnvfile tests creating container with envfile
+func (suite *PouchCreateSuite) TestCreateWithEnvfile(c *check.C) {
+	name := "TestCreateWithEnvfile"
+
+	content := "TEST1=value1"
+	validfile, err := util.TmpFileWithContent(content)
+	if err != nil {
+		c.Fatal(err)
+	}
+	defer os.Remove(validfile)
+
+	content = "TEST2="
+	invalidfile, err2 := util.TmpFileWithContent(content)
+	if err2 != nil {
+		c.Fatal(err2)
+	}
+	defer os.Remove(invalidfile)
+
+	env1 := "TEST1=value1"
+	env2 := "TEST2="
+	env3 := "TEST3=value3"
+	res := command.PouchRun("create",
+		"--name", name,
+		"--env-file", validfile,
+		"--env-file", invalidfile,
+		"-e", env3,
+		busyboxImage,
+		"top")
+	defer DelContainerForceMultyTime(c, name)
+
+	res.Assert(c, icmd.Success)
+
+	envs, err := inspectFilter(name, ".Config.Env")
+	c.Assert(err, check.IsNil)
+
+	// check if these envs are in inspect result of container.
+	if !strings.Contains(envs, env1) {
+		c.Fatalf("container env in inspect result should have %s in %s while no\n", env1, envs)
+	}
+	if !strings.Contains(envs, env2) {
+		c.Fatalf("container env in inspect result should have %s in %s while no\n", env2, envs)
+	}
+	if !strings.Contains(envs, env3) {
+		c.Fatalf("container's runtime env should have %s in %s while no\n", env3, envs)
+	}
+
+	// check if these envs are in the real container envs
+	res = command.PouchRun("start", name)
+	res.Assert(c, icmd.Success)
+
+	ret := command.PouchRun("exec", name, "env")
```

Comment:
missing `ret.Assert(c, icmd.Success)`

Reply:
I think it's no need to add this instruction.

Reply:
yes. But it can help us to know the command status. If there is output wrong, we need to know the command is correct or not. what do you think?

Reply:
I agree!